### ISSUE TITLE
redundancy between openFrameworks.cmake and CMakelists.txt

### ISSUE THREAD

**Issue by avilleret** [ofnode/of#24]
there is _a lot_ of redundancy between CMakelists.txt and openFrameworks.cmake.
Could we imagine to include the latter in the former ?

**Comment by procedural** [comment]
Yeah! Well, we could create third helper file for both of them...
